service/src/test/java/com/ge/predix/acs/policy/evaluation/cache/AbstractPolicyEvaluationCacheTest.java

```diff
@@ -89,7 +91,56 @@ public void testGetWithPolicyInvalidation() throws Exception {
         assertEquals(cachedResult.getEffect(), result.getEffect());
 
         Thread.sleep(1);
-        this.cache.resetForPolicySet(ZONE_NAME, "default");
+        this.cache.resetForPolicySet(ZONE_NAME, "policyOne");
+        assertNull(this.cache.get(key));
+    }
+
+    @Test
+    public void testGetWithResetForMultiplePolicySets() throws Exception {
+
+        PolicyEvaluationRequestV1 request = new PolicyEvaluationRequestV1();
+        request.setAction("GET");
+        request.setSubjectIdentifier("mulder");
+        request.setResourceIdentifier("/x-files");
```

Comment:
I believe there might be constants for some of these string patterns. If not, since we are using them in more than one place can we make them constants.